app/src/extensions/CustomLeftAndMainExtension.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Streunerkatzen;
+
+use SilverStripe\Admin\CMSMenu;
+use SilverStripe\Admin\LeftAndMainExtension;
+
+class CustomLeftAndMainExtension extends LeftAndMainExtension {
+    public function init() {
+        $id = "CatSubmissionsLink";
+        $title = "Katzen - Einreichungen";
```

Comment:
can we add a highlight if there's something pending? (e.g. Katzen - Einreichungen **(5)**) :D